src/app/wallet/wallet.component.ts

```diff
@@ -98,9 +98,9 @@ export class WalletComponent implements OnDestroy {
             .pipe(displayBackendErrorRx());
     }
 
-    downloadSingleReport(txHash: string, fromTxHash: string, toTxHash: string, date: Date) {
```

Comment:
If there are more than two or three parameters that can be accessed through the same object (in this case, transaction), it's better to pass the entire object. We want to avoid long parameter lists.